src/assets/styles/articles.scss

```diff
@@ -112,3 +112,23 @@
   font-style: bold !important;
   font-weight: 900;
 }
+
+.socialShare{
+  margin-top: $onepercent;
+  text-align: right;
+
+  &-twitter {
```

Comment:
Have this as 
```
&-twitter, &-facebook {
height: 5%;
width: 5%;
margin-right: 1%
}
```
This would help avoid repetition.

Reply:
Done.